lesson6/index.js

```diff
@@ -0,0 +1,27 @@
+const listData = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+const myPromise = new Promise((resolve) => {
+    setTimeout(() => {
+        resolve(listData);
+    }, 3000);
+});
+myPromise.then(() => {
+    const frag = new DocumentFragment();
+    for(const list of listData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        a.href = list.to;
+        const img = document.createElement('img');
+        img.src = list.img;
+        img.alt = list.alt;
+        const text = document.createTextNode(list.text);
+
+        a.appendChild(img);
+        a.appendChild(text);
+        li.appendChild(a);
+        frag.appendChild(li);
+    }
```

Comment:
IMO
In line 7, you pass listData as the argument of resolve, so I think you should use that in `.then`.
The current code refers directly to `listtdata`, so the result is the same whether `listData` is included in `resolve();` or not, which makes no sense.

IMO
The value passed in is tentatively named `value`, but the name of the value here does not have to be `value`, so think about it!
```suggestion
myPromise.then((value) => {
    const frag = new DocumentFragment();
    for(const list of value) {
        const li = document.createElement('li');
        const a = document.createElement('a');
        a.href = list.to;
        const img = document.createElement('img');
        img.src = list.img;
        img.alt = list.alt;
        const text = document.createTextNode(list.text);

        a.appendChild(img);
        a.appendChild(text);
        li.appendChild(a);
        frag.appendChild(li);
    }
```

Reply:
Thank you for reviewing!!

> In line 7, you pass listData as the argument of resolve, so I think you should use that in .then.
The current code refers directly to listtdata, so the result is the same whether listData is included in resolve(); or not, which makes no sense.

I understand what I was doing.
It doesn't make sense  to resolve and receive if I was referencing `listData`.

I understood that `value` is an arbitrary name, but when I looked it up, I thought that `value` was generally used in such situations.
Is it right?

I wrote using `value`.

Reply:
The fix is confirmed. Approve.
I have one request.If you can update codesandbox, it will be easier for me to review it. This will be the next time, please. Thank you very much.

Reply:
ok.Thank you very much for reviewing and comments.